### ISSUE TITLE
calendar: doesn't support tabbing out of it when used standalone 

### ISSUE THREAD

**Issue by stefan--** [angular/material#9794]
**Actual Behavior**:
-  When using md-calendar the tabindex breaks
-  A user should be able to tab through the inputs next to a md-calendar

**CodePen** (or steps to reproduce the issue): *
-  https://jsfiddle.net/08sd1hs1/1/
-  With a md-calendar tabbing is broken. If you remove the directive, it works.

**Angular Versions**: *
-  `Angular Version:` https://rawgit.com/angular/bower-material/master/angular-material.js
-  `Angular Material Version:` https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.6/angular.js

**Additional Information**:
-  `Browser Type: *` Chrome & Firefox
-  `Browser Version: *` Version 53.0.2785.116 (64-bit) & 47.0.1
-  `OS: *` Mac
-  `Stack Traces:` none

---

Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
>   Add more lines if needed.

**Comment by crisbeto** [comment]
This should be fixed when either https://github.com/angular/material/pull/9746 or https://github.com/angular/material/pull/9641 get merged.

**Comment by Splaktar** [comment]
https://github.com/angular/material/pull/9746 was merged into 1.1.2, but I tested with the jsfiddle above (which uses master) on 1.1.6 and the issue still occurs.
